warehouse/templates/accounts/register.html

```diff
@@ -70,46 +70,50 @@ <h1 class="page-title">Create an account on PyPI</h1>
           {% endif %}
         </div>
 
-        <div class="form-group">
-          <div class="split-layout">
-            <label for="password" class="form-group__label">Password</label>
-            <label for="show-password">
-              <input id="show-password" type="checkbox">&nbsp;Show passwords
-            </label>
-          </div>
-          <div>
-            {{ form.new_password(placeholder="Select a password", required="required", class_="form-group__input", autocomplete="new-password") }}
-          </div>
-          <p class="form-group__help-text">
-            Choose a strong password that contains letters (uppercase and lowercase), numbers and special characters. Avoid common words or repetition.
-          </p>
-          <p class="form-group__help-text">
-            <strong>Password strength:</strong>
-            <span class="password-strength">
+        <div data-controller="password">
+          <div class="form-group">
+            <div class="split-layout">
+              <label for="password" class="form-group__label">Password</label>
+              <label for="show-password">
+                <input data-action="click->password#showPasswords" data-target="password.showPassword"
```

Comment:
Also, I think technically the better event to bind to here is the `change` event, and since this is the default event for an input element, this can just become `data-action="password#togglePasswords"`